### DISCUSSION TITLE
how do i create my own dictionary?

### DISCUSSION THREAD

**Discussion by Tacocat263** [psoukie/zipchord#96] [Q&A]
the title

**Comment by psoukie** [comment]
Create a new text document (for example in Notepad), and fill it with records following the information about the [dictionary format](https://github.com/psoukie/zipchord#chord-dictionary).

You can also create it in Excel or another program, and then save as Tab-delimited text file -- shortcuts go into the first column, expanded text into second.

To add another shortcut to the dictionary on the fly, you can select text in any editor and hold Ctrl-C. 

Since version 2.0, you can define both a chord and a shorthand dictionary. They use the same format.

Let me know if you need more help.

**Comment by Nooby-001** [comment]
Cool, I was thinking about using Excel so the dictionaries could be sorted alphabetically.
Kinda related question, when I try to download a dictionary using "save link as" all I get is HTML, Is there a button I'm missing?
Screenshot:
https://imgur.com/a/7cedmUI

**Comment by psoukie** [comment]
To save a file from GitHub:

Click on the file you wish to download from GitHub to open the individual file. From here, right click the **Raw** button at the top of the file, select **Save Link As…**, choose the location on your computer where you want to save the file, and select **Save**. (Shamelessly copied from https://www.gitkraken.com/learn/git/github-download)

There's also a button with a copy icon that copies the contents of file into clipboard, and you could past it into a new text file and save as text, UTF-8 with BOM.
